src/Interfaces/Payment/PaymentSettledInterface.php

```diff
@@ -5,6 +5,11 @@
 namespace TrueLayer\Interfaces\Payment;
 
 use DateTimeInterface;
+use TrueLayer\Exceptions\ApiRequestJsonSerializationException;
+use TrueLayer\Exceptions\ApiResponseUnsuccessfulException;
+use TrueLayer\Exceptions\InvalidArgumentException;
+use TrueLayer\Exceptions\SignerException;
+use TrueLayer\Exceptions\ValidationException;
 use TrueLayer\Interfaces\Payment\AuthorizationFlow\ConfigurationInterface;
 
 interface PaymentSettledInterface extends PaymentRetrievedInterface
```

Comment:
does only the PaymentSettled have the methods: `refund`, `getRefund` and `getRefunds`?

Reply:
yep, indeed. The refund API only works with settled payments 👍